app/initiatives/[id]/page.tsx

```
@@ -3,13 +3,16 @@
 import React, { useEffect, useState } from 'react';
 import { useParams, useRouter } from 'next/navigation';
 import Link from 'next/link';
+import React from 'react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)